starter-project-backend/src/tests/__tests__/ArticleController.test.ts

```diff
@@ -0,0 +1,112 @@
+import app from "../../app"
+import request from "supertest"
+import * as dbHandler from "../setupdb";
+
+const agent = request.agent(app)
+
+let sampleArticle = {
+  id:"atr_9812",
+  author:{
+    firstName: "gzachew",
+    lastName: "demeke",
+    bio: "I'm blah blah"
+  },
+
+  title: "how I got to do jobs using mars",
+  content: "blah blah blah mars blah blah"
+}
+let sampleId = sampleArticle.id
+let wrongId = "ksjflksdjfkd"
+
+beforeAll(async () => {
+  await dbHandler.connect()
+})
+
+afterEach(async () => {
+    await dbHandler.clear()
+})
+
+afterAll(async () => {
+  await dbHandler.disconnect()
+})
+
+describe("GET all Article ",()=>{
+
+  test("GET all articles",async()=>{
+    const res = await agent.get("/api/articles")
+    console.log("after agent")
+    expect(res.status).toEqual(200)
+  });
+
+})
+describe("POST Article ",()=>{
+
+  test("POST article",async()=>{
+    const res = await agent.post("/api/articles").send(sampleArticle)
+    console.log("after agent")
+    expect(res.status).toEqual(200)
+  });
+
```

Comment:
add another test case for failure scenario

Reply:
fixed